src/server/pegasus_server_impl.cpp

```diff
@@ -1702,16 +1706,36 @@ ::dsn::error_code pegasus_server_impl::start(int argc, char **argv)
 
     ddebug("%s: start to open rocksDB's rdb(%s)", replica_name(), path.c_str());
 
-    auto status = rocksdb::DB::Open(rocksdb::Options(_db_opts, _data_cf_opts), path, &_db);
+    bool need_open_with_meta_cf = false;
+    // Check meta CF only when db exist.
+    if (db_exist && check_meta_cf(path, &need_open_with_meta_cf) != ::dsn::ERR_OK) {
+        derror_replica("check meta column family failed");
+        return ::dsn::ERR_LOCAL_APP_FAILURE;
```

Comment:
remove the first `::`

Reply:
Seems most existing code has `::` before `dsn`, if want to remove all leading `::`, I think we can commit another pr to do it.